_templates/_page_openshift.html.erb

```diff
@@ -175,11 +175,23 @@
       </li>
       <% if (distro_key != "openshift-origin" && distro_key != "openshift-aro" && distro_key != "openshift-acs") %>
       <span text-align="right" style="float: right !important">
-        <a href="https://github.com/openshift/openshift-docs/commits/<%= (distro_key == "openshift-enterprise") ? "enterprise-#{version}" : "dedicated-4" %>/<%= repo_path %>"><span class="material-icons-outlined" title="Page history">history</span></a>
+        <a href="https://github.com/openshift/openshift-docs/commits/<%=
+        ((distro_key == "openshift-enterprise") ? "enterprise-#{version}"
+          : (distro_key == "openshift-aro") ? "aro-work"
+          : (distro_key=="openshift-dedicated" ) ? "dedicated-4"
+          : "main" ) %>/<%= repo_path %>"><span class="material-icons-outlined" title="Page history">history</span></a>
         <%
           unless (unsupported_versions.include? version)
         %>
-        <a href="https://github.com/openshift/openshift-docs/issues/new?title=<%= (distro_key == "openshift-enterprise") ? "[enterprise-#{version}] Issue in file #{repo_path}" : ((distro_key=="openshift-dedicated" ) ? "[dedicated] Issue in file #{repo_path}" : "[online] Issue in file #{repo_path}" ) %>">
+        <a href="https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12332330&issuetype=1&components=12367614&priority=4&summary=<%=
+        ((distro_key == "openshift-enterprise") ? "[enterprise-#{version}]+Issue+in+file+#{repo_path}"
+          : (distro_key=="openshift-dedicated" ) ? "[dedicated-4]+Issue+in+file+#{repo_path}"
+          : (distro_key=="microshift" ) ? "[microshift-#{version}]+Issue+in+file+#{repo_path}"
+          : (distro_key=="openshift-aro" ) ? "[aro-#{version}]+Issue+in+file+#{repo_path}"
+          : (distro_key=="openshift-online" ) ? "[online-#{version}]+Issue+in+file+#{repo_path}"
+          : (distro_key=="openshift-rosa" ) ? "[rosa-#{version}]+Issue+in+file+#{repo_path}"
+          : (distro_key=="openshift-origin" ) ? "[origin-#{version}]+Issue+in+file+#{repo_path}"
```

Comment:
OKD hasn't migrated away from BZ, so opening issues in the OCP Jira instance doesn't make sense.